posthog/models/session_recording_event/sql.py

```diff
@@ -99,12 +100,20 @@
     ttl_period=ttl_period(),
 )
 
-KAFKA_SESSION_RECORDING_EVENTS_TABLE_SQL = lambda: SESSION_RECORDING_EVENTS_TABLE_BASE_SQL.format(
-    table_name="kafka_session_recording_events",
-    cluster=settings.CLICKHOUSE_CLUSTER,
-    engine=kafka_engine(topic=KAFKA_SESSION_RECORDING_EVENTS),
-    materialized_columns="",
-    extra_fields="",
+KAFKA_SESSION_RECORDING_EVENTS_TABLE_SQL = lambda: (
+    SESSION_RECORDING_EVENTS_TABLE_BASE_SQL.format(
+        table_name="kafka_session_recording_events",
+        cluster=settings.CLICKHOUSE_CLUSTER,
+        engine=kafka_engine(topic=KAFKA_SESSION_RECORDING_EVENTS),
+        materialized_columns="",
+        extra_fields="",
+    )
+    + " SETTINGS kafka_handle_error_mode='stream'"
```

Comment:
Could we add a `settings` field to `SESSION_RECORDING_EVENTS_TABLE_BASE_SQL` and to other sql.py files and make this setting a constant in `kafka_engine.py`?

Reasoning: It's weird how we interpolate 90% of things and then add on only this single thing at the end. Note that